guide/principles/principles.md

```diff
@@ -0,0 +1,168 @@
+- Add technical detail that is not required knowledge, or technical terms like seed phrase XPUBs, mnemonics etc.
+- Put all education up front and expect people to read and remember it
+
+---
+
+## Transparency
+
+> Open-source your products and clearly explain how they work
+
+While an open and decentralized financial system that lets people connect directly with it is great, it does put a lot of burden on the users to pick a product they trust, and that works for them. We can make this easier by freely sharing information about how our products work, what technologies they use and rely on and so forth. Let people verify that your claims are true. It is also very important to be open about what risks the user might be taking on with self-custody, under which scenarios they might lose access to their funds and best practices for avoiding this. 
+
+**Do**
+- Be open and transparent with how your product works
+- Build your product as open-source when possible
+- Let people verify, without having to trust
+- Explain what risks the user is taking on, and how best to mitigate them
+
+**Don't**
+- Make claims that are not explained or verifiable 
+
+---
+
+## Self-custody
+
+> Let users control their private keys, with no risk for seizure or freezing of funds
+
+Our existing mental models of access to digital services are usernames and passwords, controlled by a company that has custody of your funds and data. With everyone having direct access to the Bitcoin network without middlemen, there is no longer a need for people to delegate control of their funds. While it comes with greater responsibility, it is self-custody that enables the open financial system of peer-to-peer transactions.
+
+**Do**
+- Let users control their private keys and funds directly
+
+**Don't**
+- Custody funds for your users
+- Build products where the users' funds can be seized, or frozen
+
+---
+
+## Security
+
+> Provide appropriate and progressive security for all types of users
+
+Self-custody means that the end user is ultimately responsible for keeping their funds and private keys safe. They can only do that if we provide them with good, appropriate tools and awareness of best practices. It is therefore our responsibility as designers and makers to really take their security seriously and make Bitcoin products that are as secure as possible by default. We should strive for no loss of funds, regardless of whether this happens through negligence or theft. 
+
+Security is especially important when onboarding people new to Bitcoin where solutions that might be fine for advanced users might backfire for beginners. New users are likely to start by storing small amounts until they get comfortable with self-custody. A scheme suited for storing large amounts of funds is likely overkill and scary to a beginner, so the concept of progressive security is a good idea. This could let a user upgrade their security and private key management scheme as their savings grow. 
```

Comment:
> Security is especially important when onboarding people new to Bitcoin

Would be good to cross-link to `onboarding/introduction/`

Reply:
Addressed in 7cfbcba